packages/components/src/components/Trigger/Trigger.js

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import LinkIcon from '@carbon/icons-react/lib/launch/16';
+import { injectIntl } from 'react-intl';
+import React from 'react';
+import { urls } from '@tektoncd/dashboard-utils';
+import { Link } from 'react-router-dom';
+import './Trigger.scss';
+import { Table } from '@tektoncd/dashboard-components';
+
+const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
+  const tableHeaders = [
+    {
+      key: 'name',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.name',
+        defaultMessage: 'Name'
+      })
+    },
+    {
+      key: 'value',
+      header: intl.formatMessage({
+        id: 'dashboard.tableHeader.value',
+        defaultMessage: 'Value'
+      })
+    }
+  ];
+
+  let triggerParams = [];
+
+  if (trigger.params) {
+    triggerParams = trigger.params.map(param => ({
+      id: param.name,
+      name: param.name,
+      value: param.value
+    }));
+  }
+
+  let interceptorValues = [];
+
+  if (trigger.interceptor) {
+    if (trigger.interceptor.header) {
+      interceptorValues = trigger.interceptor.header.map(header => ({
+        id: header.name,
+        name: header.name,
+        value: header.value
+      }));
+    }
+  }
+
+  return (
+    <>
+      <div className="triggerresourcelinks">
+        <span className="resourcekind">TriggerBinding</span>
+        <Link
+          to={urls.triggerBindings.byName({
+            namespace: eventListenerNamespace,
+            triggerBindingName: trigger.binding.name
+          })}
+        >
+          <span className="truncate-text-end">{trigger.binding.name}</span>
```

Comment:
should have `title={trigger.binding.name}` on the span to ensure the user can still view the full name on hover when truncated